### DISCUSSION TITLE
Select notebook kernel by default

### DISCUSSION THREAD

**Discussion by adijo** [jupyterlite/jupyterlite#1123] [Q&A]
Hi, I was trying to find a way to select Notebook Python (Pyodide) by default when the page is loaded. Thanks in advance for the help!

![Screenshot 2023-08-14 at 1 27 14 PM](https://github.com/jupyterlite/jupyterlite/assets/3933447/07f1461d-5327-41db-a367-8de9e586b7a6)

**Comment by jtpio** [comment]
@adijo maybe the simplest would be to first add some contents to your JupyterLite deployment, and then create a link that opens a notebook by default? Following https://jupyterlite.readthedocs.io/en/latest/howto/content/share.html.
